backend/app/controllers/spree/admin/products_controller.rb

```diff
@@ -73,6 +68,15 @@ def clone
 
       private
 
+      def split_params
+        if params[:product][:taxon_ids].present?
```

Comment:
this could probably be dried up a bit since the logic is exactly the same, not strictly speaking necessary though